util/lksconfig

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Allows to read/write the configuration of Andor Shamrock spectrograph
+'''
```

Comment:
To be consistent with PEP9, can you use triple double quotes for docstrings? https://peps.python.org/pep-0257/#what-is-a-docstring

Reply:
Almost all the files already starts with single quoted docstrings but would be a good indeed to update them all whenever we're working on a file that has single quoted docstring. So that the inconsistency would phase out over time!